e2etest/pkg/service/create.go

```diff
@@ -43,17 +56,15 @@ func Delete(cs clientset.Interface, svc *corev1.Service) {
 	framework.ExpectNoError(err)
 }
 
-func tweakServicePort(svc *v1.Service) {
-	if TestServicePort != 80 {
-		// if TestServicePort is non default, then change service spec.
-		svc.Spec.Ports[0].TargetPort = intstr.FromInt(int(TestServicePort))
+func TweakServicePort(port int) func(svc *v1.Service) {
+	return func(svc *v1.Service) {
+		svc.Spec.Ports[0].TargetPort = intstr.FromInt(int(port))
```

Comment:
port is already int. Is the casting necessary here?

Reply:
Nope, fixed